lib/child_process.js

```diff
@@ -440,9 +445,11 @@ function normalizeSpawnArguments(file, args, options) {
   }
```

Comment:
`normalizeSpawnArguments()` also has a call to `Object.assign()` that clones the options. We could probably move that line up before the option validation and use `Object.create(null)` there too.